src/main/scala/com/ossuminc/riddl/plugins/idea/files/RiddlTokenizer.scala

```diff
@@ -49,27 +45,178 @@ object RiddlTokenizer {
     }
   }
 
-  val keywords: Seq[String] = Seq("acquires", "adaptor", "all", "any", "append", "application", "author",
-    "become", "benefit", "brief", "briefly", "body", "call", "case", "capability", "command", "commands",
-    "condition", "connector", "constant", "container", "contains", "context", "create", "described", "details",
-    "direct", "presents", "do", "domain", "else", "email", "end", "entity", "epic", "error", "event", "example",
-    "execute", "explained", "field", "fields", "file", "flow", "focus", "foreach", "form",
-    "function", "graph", "group", "handler", "if", "import", "include", "index", "init", "inlet", "inlets",
-    "input", "invariant", "items", "many", "mapping", "merge", "message", "morph", "name", "on", "one", "organization",
-    "option", "optional", "options", "other", "outlet", "outlets", "output", "parallel", "pipe", "plant", "projector",
-    "query", "range", "reference", "remove", "replica", "reply", "repository", "requires", "required", "record",
-    "result", "results", "return", "returns", "reverted", "router", "saga", "schema", "selects", "send", "sequence",
-    "set", "show", "shown", "sink", "source", "split", "state", "step", "stop", "story", "streamlet", "table", "take",
-    "tell", "term", "then", "title", "type", "url", "updates", "user", "value", "void", "when", "where")
+  val keywords: Seq[String] = Seq(
```

Comment:
This list is missing some new keywords. Please check the riddl sources again to find the exact delta, but by memory this is what I found missing:
`with`, `nebula`, `relationship`, `attachment`

There are likely others.

Reply:
Also, there may have been some old keywords dropped out of usage. So look for things in your list, but not in RIDDL source list.